internal/cmd/connect/connect.go

```diff
@@ -109,57 +110,92 @@ argument:
 				return errors.New("database branch is not ready yet")
 			}
 
+			pw, err := passwordutil.New(ctx, client, passwordutil.Options{
+				Organization: ch.Config.Organization,
+				Database:     database,
+				Branch:       branch,
+				Role:         role,
+				Name:         passwordutil.GenerateName("pscale-cli-connect"),
+				TTL:          24 * time.Hour, // TODO: use shorter TTL, but implement refreshing
```

Comment:
Reading this TODO comment, with no password refreshing, I don't think we can implement/ship this as-is.